### PR TITLE
Add boolean to experience models for fulfilling education requirements

### LINKED ISSUE #2839: Add an "education_justification" field to Experience models
~Create a table of justification types. Currently FORMAL and COMBINATION. Create a lookup table between all experience types and the justification types. This means that any Experience object can be flagged as a FORMAL justification, a COMBINATION justification, both, or neither.~

All the new Experience models need a single boolean field to represent whether they satisfy education requirements or not.

### PR DESCRIPTION
**Pull request by gggrant**
Resolves #2839.

### Notes:
- Merge #2981 first.
- Default `true` for the education table?

### CODE REVIEW

File: database/migrations/2020_02_21_233331_add_experience_education_table.php [hunk 1]
```diff
@@ -26,6 +26,7 @@ public function up()
             $table->boolean('has_blockcert')->nullable();
             $table->integer('experienceable_id')->unsigned()->index();
             $table->string('experienceable_type');
+            $table->boolean('is_education_requirement')->default(true);
```
Comment (tristan-orourke) [inline database/migrations/2020_02_21_233331_add_experience_education_table.php]: Hmm. Good question, but I think we should leave it default(false) for consistency, and leave it up to the user. We can always suggest it in the UI if we want to.

File: app/Models/ExperienceAward.php [hunk 2]
```diff
@@ -27,18 +28,20 @@ class ExperienceAward extends BaseModel
...
         'award_recipient_type_id',
         'issued_by',
         'award_recognition_type_id',
-        'awarded_date'
+        'awarded_date',
+        'is_education_requirement'
     ];
 
-    protected $table = 'experiences_award';
+    protected $table = ['experiences_award'];
```
Comment (tristan-orourke) [inline app/Models/ExperienceAward.php]: Oh, I forgot one more comment! These table statements should be strings, not arrays (ie whats already in dev).

Comment (tristan-orourke) [reply app/Models/ExperienceAward.php]: I'll fix it, and merge this in

**Comment by gggrant** [comment]
> Normally its a bad idea to edit migrations created in a different PR, because its hard to know whether those migrations have been run on any of our servers yet (in which case the updated migration won't run the second time).
> If you're sure the experience table migrations haven't run anywhere yet, then it's okay this time.

Got it. I'm sure they haven't run in this case.

File: app/Models/ExperienceAward.php (hunk 2 above)
Comment (tristan-orourke) [reply app/Models/ExperienceAward.php]: Oh sweet, nvm.

**Pull request merged**